connmgr.go

```diff
@@ -51,6 +51,15 @@ func (d *Daemon) doConnManager(req *pb.Request) *pb.Response {
 		d.host.ConnManager().TrimOpenConns(ctx)
 		return okResponse()
 
+	case pb.ConnManagerRequest_DISCONNECT_PEER:
```

Comment:
Seems a bit of a misnomer to place this operation in the namespace of `ConnManager`? Why not put it in the same namespace as `CONNECT`?

Reply:
Ok, I will move it to top-level.